config/apache2-ironic-api.conf.j2

```diff
@@ -0,0 +1,52 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+Listen 6385
+
+{% if env.LISTEN_ALL_INTERFACES | lower == "true" %}
+ <VirtualHost *:6385>
+{% else %}
+ <VirtualHost {{ env.IRONIC_IP }}:6385>
```

Comment:
Thanks! This is better. One small nit, if IRONIC_IP is IPv6, this won't get wrapped correctly, e.g it'd need to be [fd00::2]:6385.  I think you can use the `env.IRONIC_URL_HOST` instead, which is built in the ironic common.sh include.

Reply:
Do you mean change `env.IRONIC_IP` to `env.IRONIC_URL_HOST` in line 18?